tailwind.config.js

```diff
@@ -1,31 +1,31 @@
 // tailwind.config.js
 module.exports = {
-  darkMode: 'class',
+  darkMode: "class",
   content: [
-    './src/**/*.{js,jsx,ts,tsx}',
-    './src/pages/**/*.{js,jsx,ts,tsx}',
-    './src/components/**/*.{js,jsx,ts,tsx}',
-    './src/templates/**/*.{js,jsx,ts,tsx}',
+    "./src/**/*.{js,jsx,ts,tsx}",
+    "./src/pages/**/*.{js,jsx,ts,tsx}",
+    "./src/components/**/*.{js,jsx,ts,tsx}",
+    "./src/templates/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
     colors: {
-      white: '#ffffff',
-      red: '#cc091c',
+      white: "#ffffff",
+      red: "#8A2A2B",
       grey: {
-        light: '#ecf0f1',
-        default: '#cbd5e0',
```

Comment:
i can play around with these grays, not totally sure where each one is used, so maybe ii can use the "default" with one of the new grays instead

Reply:
Yeah it looks like on the pages the lighter gray backgrounds all went with the new darker palette. And because of that the blue and darker red might seem a bit too dark?

Reply:
yea, seems a bit dark for the foreground 🤔 . I tried to play with it a bit and we can think on it if we'd like, but some of these may require changes to existing components.  I'm noticing some of our "card-like" grid UIs use different colors, styling and markup, so maybe we could look into this as a future issue.  We could send the preview link to more folks to get feedback, but I think if these look ok and our fix for light-blue-on-white subtitle text is agreeable, then we can go with this for now 👍